utils/netns/netns.go

```diff
@@ -59,3 +71,28 @@ func IsVethExists(vethHostName string) (bool, error) {
 	}
 	return false, nil
 }
+
+// GetLinkLocalAddr Get the IPv6 link local address of interfaceName
+func GetLinkLocalAddr(interfaceName string) net.IP {
+	link, err := netlink.LinkByName(interfaceName)
+	if err != nil {
+		log.Fatal(err)
+	}
+	addrs, err := netlink.AddrList(link, netlink.FAMILY_V6)
+	if err != nil {
+		log.Fatal(err)
+	}
+	var linkLocalAddr net.IP
+	for _, addr := range addrs {
+		if addr.Scope == syscall.RT_SCOPE_LINK {
+			linkLocalAddr = addr.IP
+			break
+		}
+	}
+	if linkLocalAddr == nil {
+		log.Warn(fmt.Sprintf("No IPv6 link local address found for interface: %s", interfaceName))
+		// return dummy link local (derived from MAC = EE:EE:EE:EE:EE:EE)
```

Comment:
Is this really safe to do? I don't think it will work as there is nothing to respond to ARP requests for this IP